packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -249,11 +249,9 @@ const ProgramCard = ({
       <IconWrapper {...programStateConfig[state]}>
         {mediaIcons.audio}
       </IconWrapper>
+
       <DurationWrapper dir={dir} dateTime={duration}>
-        <VisuallyHiddenText>
-          {` ${durationLabel} `}
-          {`${formatDuration(duration, getDurationFormat(duration, ','))} `}
```

Comment:
@greenc05 Do you remember by any chance if we had to add the duration within the visually hidden text?

Reply:
Yes we need that, as it's in a different format with the comma, like our media durations, as that is less likely to read out as a time format e.g. 2 o'clock

Reply:
In that case, shall we apply `aria-hidden` to the visual duration instead, so it is not read out twice?

Reply:
I thought that's what we had already done?

Reply:
I actually think this bug is being caused by some of our CSS

Reply:
As the bug isn't present wheh there is no CSS...

Reply:
True, the `DurationTextWrapper` has it already.
`const DurationTextWrapper = styled.span.attrs({ 'aria-hidden': 'true' })``;`

Reply:
Yes we already have that @DenisHdz